package.json

```diff
@@ -61,6 +61,8 @@
     "@opentelemetry/tracing": "0.8.3",
     "@sentry/browser": "5.17.0",
     "@sentry/node": "5.17.0",
+    "@sentry/react": "^5.22.3",
+    "@sentry/tracing": "^5.22.3",
```

Comment:
Interesting, are you the first to use sentry in a react app?

Reply:
Apparently 🤷‍♂️  But Sentry is the official way to go according to 👉 https://github.com/island-is/handbook/blob/master/docs/adr/0005-error-tracking-and-monitoring.md